.github/workflows/gen-docs.yaml

```diff
@@ -0,0 +1,15 @@
+name: readthedocs/actions
```

Comment:
We probably should add license in header to all new files (wherever possible) cc @vishesh92 @DaanHoogland

Reply:
Missed that. Let me create another PR for this.